src/credentials/sso/shared.ts

```diff
@@ -0,0 +1,29 @@
+/*!
```

Comment:
can this live in sso/model.ts or somewhere else? we have 5+ `sso/*` modules, that is a lot for this one topic